superduperdb/base/build.py

```diff
@@ -158,7 +158,9 @@ def build_compute(compute):
     if compute.split('://')[0] == 'ray':
         return RayComputeBackend(compute)
 
-    return LocalComputeBackend()
+    compute = LocalComputeBackend()
+
+    logging.info("Connecting to compute client:", compute.name)
```

Comment:
We need similar logging for all the connection methods (within `if`).

Also the "connecting" message should go before the connection code (not after)